IpmiFeaturePkg/IpmiFeaturePkg.dsc

```diff
@@ -85,6 +85,16 @@
   ReportStatusCodeLib|MdeModulePkg/Library/SmmReportStatusCodeLib/SmmReportStatusCodeLib.inf
   HobLib|MdePkg/Library/DxeHobLib/DxeHobLib.inf
 
+[LibraryClasses.X64]
+  RngLib|MdePkg/Library/BaseRngLib/BaseRngLib.inf
+!if $(TOOL_CHAIN_TAG) == VS2017 or $(TOOL_CHAIN_TAG) == VS2015 or $(TOOL_CHAIN_TAG) == VS2019 or $(TOOL_CHAIN_TAG) == VS2022

```

Comment:
```suggestion
$(TOOL_CHAIN_TAG) == VS2019 or $(TOOL_CHAIN_TAG) == VS2022

```

Reply:
https://github.com/microsoft/mu_basecore/blob/release/202208/MdeModulePkg/MdeModulePkg.dsc#L135

It moved to only supporting VS2022 and VS2019, as referenced above.